docs/src/main/asciidoc/continuous-testing.adoc

```diff
@@ -137,13 +138,13 @@ state.
 [l] - Toggle live reload::
 This is not directly related to testing, but allows you to turn live reload on and off.
 
-[s] - Force live reload scan::
+[s] - Force restart with any changes::
 This will force a scan for changed file, and will perform a live reload if anything has changed. This will still work even
 if live reload is disabled.
```

Comment:
Does it force a scan? What happens if no changes were made?

Reply:
Actually it will always reload now, so I need to update this